docs/proposals/20200416-RollingReleasesBranchingStrategy.md

```diff
@@ -0,0 +1,79 @@
+# Proposal: RollingReleasesBranchingModel
+
+Author(s): Andrew Rynhard
+
+## Abstract
+
+[A short summary of the proposal.]
+
+## Background
+
+[An introduction of the necessary background and the problem being solved by the proposed change.]
```

Comment:
some goals we captured yesterday during the call?

I would also add that no commit might be tagged with two release tags, and that `alpha` tags are all on `master` branch commits, `beta` and `stable` only on release branches.

Reply:
Ahh yes, good one.